pipeline_dp/accumulator.py

```diff
@@ -239,6 +239,54 @@ def compute_metrics(self):
         return dp_computations.add_noise_vector(self._vec_sum, self._params)
 
 
+class VectorMeanAccumulator(Accumulator):
+    _vec_sum: np.ndarray
```

Comment:
I missed this in the previous CL, this means that _vec_sum/_params are class level variables and in __init__ instance level _vec_sum/_params created. Please remove (and in `VectorSummationAccumulator` as well)

Reply:
Agreed, done.